### PR TITLE
Add French language tag

### PR DESCRIPTION
**Pull request by fpigerre**
Hello!

This pull request adds a French language tag to the`languages.yml` file.

Thanks,
psgs 🌴

### CODE REVIEW

**Review by Jawshy — COMMENTED** [review]
Thanks for doing this. :+1:

File: _data/languages.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ de:
 en:
   work_tweet: Security is important, @TWITTERHANDLE. We'd like it if you supported two factor auth.
   progress_tweet: Thanks for working on support for two factor auth, @TWITTERHANDLE!
+fr:
+  work_tweet: La sécurité est importante @TWITTERHANDLE. Nous recommandons un système d'identification à deux facteurs.
+  progress_tweet: Merci d'avoir commencé à mettre en place un système d'identification à deux facteurs, @TWITTERHANDLE!
 sv:
   work_tweet: Säkerhet är viktigt, @TWITTERHANDLE. Vänligen lägg till two factor authentication på er hemsida.
   progress_tweet: Tack för att ni arbetar på two factor authentication, @TWITTERHANDLE!
-
```
Comment (Carlgo11) [inline _data/languages.yml:12]: needs to have an empty line at the end of the file.

Comment (Jawshy) [reply _data/languages.yml:12]: I downloaded the file and there is an empty line at the end.

Comment (Carlgo11) [reply _data/languages.yml:12]: ah, looks like there is. GitHub just didn't show it. Never mind then

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**